config.lua

```diff
@@ -3,6 +3,7 @@ Config.MaxWidth = 5.0
 Config.MaxHeight = 5.0
 Config.MaxLength = 5.0
 Config.DamageNeeded = 100.0
+Config.Headshot = true
```

Comment:
i think it would be better if you change it to `Config.NoHeadshotOneshot` or something other than this, it could be a little bit confusing for normal users that what Headshot in config means

Reply:
as @ali-exacute stated that would be a good idea, the clearer for the end-users the better.